app/src/main/java/org/sopt/sample/ui/main/search/ReqresAdapter.kt

```diff
@@ -0,0 +1,46 @@
+package org.sopt.sample.ui.main.search
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.DiffUtil
+import androidx.recyclerview.widget.ListAdapter
+import androidx.recyclerview.widget.RecyclerView
+import com.bumptech.glide.Glide
+import org.sopt.sample.data.remote.entity.reqres.ResponseReqresDTO
+import org.sopt.sample.databinding.ItemReqresBinding
+
+class ReqresAdapter : ListAdapter<ResponseReqresDTO.Data, ReqresAdapter.ReqresViewHolder>(ReqresComparator()) {
```

Comment:
```suggestion
class ReqresAdapter : ListAdapter<ResponseReqresDTO.Data, ReqresAdapter.ReqresViewHolder>(ReqresComparator) {
```

ReqresComparator -> object로 사용해도 될 것 같음